### DISCUSSION TITLE
함수 선언과 함수 표현식의 스코프 차이는 무엇인가요?

### DISCUSSION THREAD

**Discussion by wogus216** [diving-javascript/you-dont-know-js-yet#47] [1. 스코프]
답변

**Comment by lledellebell** [comment]
<details>
<summary>답변</summary>
함수 선언과 함수 표현식은 모두 함수를 정의하는 방법이지만, 함수 선언은 전역 스코프에 속하고 함수 정의 전에 호출할 수 있는 반면, 함수 표현식은 변수 스코프에 속하고 함수 정의 후에만 호출할 수 있습니다.
</details>
